### PR TITLE
Hds 171 updated koros accessibility tab

### PR DESCRIPTION
**Pull request by minnas**
## Description

Adds correct content to Koros Accessibility tab

## Motivation and Context

[HDS-1714](https://helsinkisolutionoffice.atlassian.net/browse/HDS-1714)

## How Has This Been Tested?

Tested locally

### CODE REVIEW

File: site/src/docs/components/koros/accessibility.mdx [hunk 1]
```diff
@@ -10,5 +10,7 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
 ## Accessibility
 
 ### Pay attention to
-- HDS Footer is designed to support many different brand colours. When customising colours, refer to <InternalLink href="/foundation/design-tokens/colour">colour guidelines</InternalLink> to ensure accessibility.
-- **When designing the footer link hierarchy, always think about keyboard and screen reader users.** Remember that keyboard users need to manually navigate to each element.
+
+- HDS Koros should be centered and have at least 2 full wave motif tips visible
+- Do not use Koro shapes as an accent element (not for example, for active element)
+- It's recomended to stick one with one Koro size and shape though the whole service
```
Comment (mrTuomoK) [inline site/src/docs/components/koros/accessibility.mdx:16]: I'd rephrase maybe a bit -> ...recommended to use the same Koro size and shape... though -> throughout (or across etc.)

**Review by mrTuomoK — APPROVED** [review]
Fine by me, maybe also another review by someone who knows the Koros would be nice.

File: site/src/docs/components/koros/accessibility.mdx [hunk 2]
```diff
@@ -10,5 +10,7 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
 ## Accessibility
 
 ### Pay attention to
-- HDS Footer is designed to support many different brand colours. When customising colours, refer to <InternalLink href="/foundation/design-tokens/colour">colour guidelines</InternalLink> to ensure accessibility.
-- **When designing the footer link hierarchy, always think about keyboard and screen reader users.** Remember that keyboard users need to manually navigate to each element.
+
```
Comment (laurakarhu) [inline site/src/docs/components/koros/accessibility.mdx:13]: These three are not really accesibility concerns, but should be included in the general principles

Comment (laurakarhu) [inline site/src/docs/components/koros/accessibility.mdx:13]: For accessibility I would suggest:
Koros are decorative items and should be used in background and hidden from screen readers. See e.g. [Hero](https://hds.hel.fi/components/hero) and [Footer](https://hds.hel.fi/components/footer).
Note that in this documentation the koros have alt texts as they present the component and are therefore important content.

Koros are not an interactive element and do not receive focus.

**Review by laurakarhu — CHANGES_REQUESTED** [review]

File: site/src/docs/components/koros/accessibility.mdx [hunk 3]
```diff
@@ -10,5 +10,7 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
...
+- Koros are decorative items and should be used in background and hidden from screen readers. See e.g. <InternalLink href="/components/hero">Hero</InternalLink> and <InternalLink href="/components/footer">Footer</InternalLink>.
+- **Note that in this documentation the koros have alt texts as they present the component and are therefore important content.**
+- Koros are not an interactive element and do not receive focus.
```
Comment (harriplappalainen) [inline site/src/docs/components/koros/accessibility.mdx:16]: Koros is in plural, so this should be: "Koros are not interactive elements and do not receive focus."

File: site/src/docs/components/koros/index.mdx [hunk 4]
```diff
@@ -30,6 +30,9 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
 - Use koros as part of a background. Do not create new visual elements using koros.
 - It is allowed to animate Koros shapes (not included in the HDS yet).
 - A koro shape can be rotated in 45 degree increments. For more information see the [Angled koro variant documentation](#angled) below.
+- Koros should be centered and have at least 2 full wave motif tips visible
```
Comment (harriplappalainen) [inline site/src/docs/components/koros/index.mdx:33]: Missing dot at the end of sentence.

File: site/src/docs/components/koros/index.mdx [hunk 5]
```diff
@@ -30,6 +30,9 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
...
+- Do not use Koro shapes as an accent element (not for example, for active element)
```
Comment (harriplappalainen) [inline site/src/docs/components/koros/index.mdx:34]: - Koro shapes is in plural, so it should be: "--Koro shapes as accent elements--". 
- The end of sentence might "flow" better if restructured like this: "- Do not use Koro shapes as accent elements, for example to display active elements."
- Remember a dot at the end

File: site/src/docs/components/koros/index.mdx [hunk 6]
```diff
@@ -30,6 +30,9 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
...
+- It's recomended to use one Koro size and shape through the whole service
```
Comment (harriplappalainen) [inline site/src/docs/components/koros/index.mdx:35]: Typos:
- recommended
- dot at the end of sentence

**Review by harriplappalainen — COMMENTED** [review]
Nice fix! 🏅 I have some grammar suggestions in the comments.

**Review by harriplappalainen — APPROVED** [review]
Looks great 👍

**Comment by minnas** [comment]
I created new branch and PR for the changes.
